Sources/PointFree/Emails/NewBlogPostEmail.swift

```diff
@@ -30,51 +31,51 @@ let newBlogPostEmail = simpleEmailLayout(newBlogPostEmailContent)
 
 let newBlogPostEmailContent = View<(BlogPost, String?)> { post, announcement -> Node in
   emailTable([style(contentTableStyles)], [
-    tr([
-      td([valign(.top)], [
-        div(
-          [`class`([Class.padding([.mobile: [.all: 0], .desktop: [.all: 2]])])],
-          announcementView.view(announcement)
-        ),
-
-        div([`class`([Class.padding([.mobile: [.all: 0], .desktop: [.all: 2]])])], [
-          a([href(url(to: .blog(.show(post))))], [
-            h3([`class`([Class.pf.type.responsiveTitle3])], [.text(post.title)]),
-            ]),
-            p([text(post.blurb)])
-          ]
-          + (
-            post.coverImage.map {
-              [
-                p([`class`([Class.padding([.mobile: [.topBottom: 2]])])], [
-                  a([href(url(to: .blog(.show(post))))], [
-                    img([src($0), alt(""), style(maxWidth(.pct(100)))])
-                    ])
-                  ]),
-                ]
-              } ?? [])
-          + [
-            a(
-              [
-                href(url(to: .blog(.show(post)))),
-                `class`(
-                  [
-                    Class.pf.colors.link.purple,
-                    Class.pf.colors.fg.purple,
-                    Class.pf.type.body.leading
-                  ]
-                )
-              ],
-              ["Read the full post…"]
-            )
-          ]),
-
-        div(
-          [`class`([Class.padding([.mobile: [.all: 0], .desktop: [.all: 2]])])],
-          hostSignOffView.view(unit)
-        )
-        ])
-      ])
+//    tr([
```

Comment:
was having trouble getting this to type check so just killed the whole thing for now